packages/appstore/src/components/add-options/add-options.scss

```diff
@@ -15,11 +17,18 @@
         & > p {
             color: $color-red;
             margin-left: 0.8rem;
+            font-weight: 700;
```

Comment:
as we talked earlier `DO NOT` add these styles is css, use them as props in the `Text` component

Reply:
@mahdiyeh-fs  This Text component is not the one we are using the other places, I haven't seen any size and line height attribute on it.

Reply:
let's replace that with Text  from '@deriv/components and add these props to it.